### PR TITLE
fixed naming of PromethION positions

### PR DESCRIPTION
**Pull request by djcooke**
* corrects names of positions that are active at OICR
* adds positions that are not functional at OICR
* deletes positions that were incorrectly named according to both sources

Will run on stage after approved and prod after release

### CODE REVIEW

**Comment by djcooke** [comment]
Er.. this is not replayable. Will fix

File: sqlstore/src/main/resources/db/migration/V8501__fix_promethion_pos.sql [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+  DECLARE promethion bigint(20);
+  SELECT instrumentModelId INTO promethion FROM InstrumentModel WHERE alias = 'PromethION';
+  
+  IF NOT EXISTS (SELECT 1 FROM InstrumentPosition WHERE instrumentModelId = promethion AND alias = iName) THEN
+    INSERT INTO InstrumentPosition (instrumentModelId, alias) VALUES (promethion, iName);
+  END IF;
+END//
+DELIMITER ;
+
+INSERT INTO InstrumentPosition (instrumentModelId, alias) VALUES
```
Comment (callunity) [inline sqlstore/src/main/resources/db/migration/V8501__fix_promethion_pos.sql]: Ah. This is the offending line.

Comment (djcooke) [reply sqlstore/src/main/resources/db/migration/V8501__fix_promethion_pos.sql]: Oops. Fixed

File: sqlstore/src/main/resources/db/migration/V8501__fix_promethion_pos.sql [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+    INSERT INTO InstrumentPosition (instrumentModelId, alias) VALUES (promethion, iNewName);
+  END IF;
+END//
+DELIMITER ;
+
+CALL TempFixPos('P101_0', '1-A1-D1');
+CALL TempFixPos('P105_0', '1-A3-D3');
+CALL TempFixPos('P109_0', '1-A5-D5');
+
+CALL TempFixPos('P101_1', '1-E1-D1');
```
Comment (callunity) [inline sqlstore/src/main/resources/db/migration/V8501__fix_promethion_pos.sql]: `1-E1-D1` -> `1-E1-H1`; same applies for all in `TempFixPos`

**Pull request merged**